features/step_definitions/flipkart.rb

```diff
@@ -0,0 +1,46 @@
+require 'selenium-webdriver'
+require 'test-unit'
+
+  caps = {
+  	'bstack:options' => {
+  		"os" => "Windows",
+  		"osVersion" => "10",
+  		"local" => "false",
+  		"consoleLogs" => "info",
+  		"networkLogs" => "true",
+  		"seleniumVersion" => "3.5.2",
+  	},
+  	"browserName" => "IE",
+  	"browserVersion" => "11.0",
+  }
+
+
+
+  driver = Selenium::WebDriver.for(:remote,
+    :url => "http://" + ENV['USERNAME'] + ":" + ENV['PASSWORD'] + "@hub-cloud.browserstack.com/wd/hub",
+    :desired_capabilities => caps)
```

Comment:
Implement using cucumber hooks in support folder

Reply:
Done